src/main/groovy/life/qbic/datamodel/dtos/business/ProductId.groovy

```diff
@@ -0,0 +1,67 @@
+package life.qbic.datamodel.dtos.business
+
+import groovy.transform.CompileStatic
+
+/**
+ * A DTO describing Product Identifiers
+ *
+ * The ProductId should be used as a general class describing identifiers like the {@link OfferId} and the {@link CostEstimateId}.
+ * It consists of a fixed Part depending on the product category and an Integer based versioning
+ *
+ * @since: 1.13.0
+ *
+ */
+
+@CompileStatic
+class ProductId {
+
+    /**
+     * The type of the identifier is defined by the implementing identifier
+     */
+    final private String type
+
+    /**
+     * Version of the identifier
+     */
+    final private String version
```

Comment:
Is it the version or rather the actual id? I would avoid using the name version here since it is very missleading

Reply:
Good point 👍